docs/roman/assign_wcs/main.rst

```diff
@@ -6,14 +6,14 @@ Description
 It associates a World Coordinate System (WCS) object with each science exposure.
 Note that no fitting is performed in this step; it only creates a WCS object that
 transforms positions in the ``detector`` frame to positions in the ``world``
-coordinate frame (ICRS) based on the telescope pointing and reference files provided by CRDS.
+coordinate frame (ICRS) based on the telescope pointing and reference files in CRDS.
 The constructed WCS object can be accessed as an attribute of the ``meta`` object
 when the file is opened as a data model. The forward direction of the transforms is
 from detector to world coordinates and the input positions are 0-based.
 
 ``romancal.assign_wcs`` uses `GWCS <https://github.com/spacetelescope/gwcs>`__ -
 a package for managing the World Coordinate System of astronomical data.
-It expects to find the basic WCS keywords in the
+It expects to find a few basic WCS keywords in the
```

Comment:
Should we list the keywords here? or just say they are the ones below?

Reply:
They are listed above this paragraph so I consider them "defined".